README.md

```diff
@@ -150,15 +148,18 @@ Only information in the manifest and below can be trusted.
 
 ### Main content
 
-The main content of a package is a sequence of HTTP request/response pairs
-holding the manifests of sub-packages and the resources in the package and all
-of its sub-packages. These resources can appear in any order.
+The main content of a package is an index of HTTP requests pointing to HTTP
+responses. These request/response pairs hold the manifests of sub-packages and
+the resources in the package and all of its sub-packages. Both the requests and
+responses can appear in any order, usually chosen to optimize loading while the
+package is streamed.
 
 ```cddl
-$section-name /= "content"
-$$section //= ("content" =>
-  {* resource-key => [response-headers: http-headers, body: bstr]}
-)
+$section-name /= "indexed-content"
+$$section //= ("indexed-content" => [
+  index: [* [resource-key, offset, ? length: uint] ],
+  responses: [* [response-headers: http-headers, body: bstr]],
```

Comment:
I don't strictly need the nested array here: it could just be a list of alternating headers and bodies.